packages/webdriverio/src/index.ts

```diff
@@ -0,0 +1,230 @@
+import type { RunOptions, AxeResults } from 'axe-core';
+import { source } from 'axe-core';
+import * as assert from 'assert';
+import * as cssesc from 'cssesc';
+import type {
+  Options,
+  CallbackFunction,
+  BrowserObject,
+  Element
+} from './types';
+import {
+  analyzePage,
+  isWebdriverClient,
+  normalizeContext,
+  logOrRethrowError
+} from './utils';
+
+export default class AxeBuilder {
```

Comment:
Is it "safe" to change the export style here? The previous version used CommonJS-style exports I believe.